pkg/kubectl/cmd/util/openapi/openapi.go

```diff
@@ -92,7 +96,23 @@ func (p *Path) Len() int {
 }
 
 func (p *Path) String() string {
-	return strings.Join(p.Get(), ".")
+	return strings.Join(p.Get(), "")
```

Comment:
We used to append raw keys to the Path, now we use [`FieldPath`](https://github.com/kubernetes/kubernetes/pull/50160/files#diff-a26f1a64464c16ea39d47654abbcdb26R111) that automatically adds the `.`